src/web/components/ClientSideTokenGeneration/CstgAddDomainDialog.spec.tsx

```diff
@@ -61,21 +61,12 @@ describe('CstgDomainAddDomainDialog', () => {
 
     await submitDialog();
 
-    await waitFor(() => {
```

Comment:
What's the reason for removing this?

Reply:
Removed this because the change from URL to root-level domain is no longer happening on the front-end, but on the backend instead, so this test was originally to make sure all domains sent to the back-end were root-level (i.e. http://test4.com turning to test4.com ).  I'm thinking maybe we don't even need to test altogether anymore

Reply:
Put it back